src/magefile.go

```diff
@@ -165,13 +176,13 @@ func Test386() error {
 // Run tests
 func Test() error {
 	env := map[string]string{"GOFLAGS": testGoFlags()}
-	return runCmd(env, goexe, "test", "./...", "-tags", buildTags())
+	return runCmd(env, goexe, "test", "./...", buildFlags(), "-tags", buildTags())
 }
 
 // Run tests with race detector
 func TestRace() error {
 	env := map[string]string{"GOFLAGS": testGoFlags()}
-	return runCmd(env, goexe, "test", "-p 1", "-race", "./...", "-tags", buildTags())
```

Comment:
We added this change specifically for Gotham. We shouldn't remove it.

Reply:
re-adding "-p 1"